docs/ReferenceResolution.md

```diff
@@ -65,27 +68,28 @@ Steps for adding a new package dependency to an existing project. Let's say I'm
         </ProductDependencies>
         ```
 
-       If you don't know the commit hash of the source code used to produce "0.0.1-beta-1", you can use `000000` as a placeholder for `Sha`
-       as its value is unimportant and will be updated the next time the bot runs.
+        If you don't know the commit hash of the source code used to produce "0.0.1-beta-1", you can use `000000` as a
+        placeholder for `Sha` as its value will be updated the next time the bot runs.
 
-        If the new dependency comes from dotnet/CoreFx, dotnet/code-setup or dotnet/extensions, add a
+        If the new dependency comes from dotnet/runtime and you are updating dotnet/aspnetcore-tooling, add a
         `CoherentParentDependency` attribute to the `<Dependency>` element as shown below. This example indicates the
-        dotnet/CoreFx dependency version should be determined based on the build that produced the chosen
-        Microsoft.NETCore.App. That is, the dotnet/CoreFx dependency and Microsoft.NETCore.App should be
-        coherent.
+        dotnet/runtime dependency version for System.Banana should be determined based on the dotnet/aspnetcore build
```

Comment:
aspnetcore -> aspnetcore-tooling

Reply:
Actually, we don't depend on tooling anymore - I think we can get rid of all CPDs in this repo now that the only repo below us is dotnet/runtime (@mmitche please confirm)

Reply:
No; runtime bits flow into aspnetcore and from there into aspnetcore-tooling. This part of the document describes the CPD setup needed to keep that coherent.

Reply:
Yes language here is correct.